### DISCUSSION TITLE
Forwarding From index.php to another page if there is no GET variable

### DISCUSSION THREAD

**Discussion by mnazw** [YOURLS/YOURLS#3220] [Q&A]
Hello, first, sorry for my bad english, it's not my mother tounge. I've tried to install YOURLS on my website subdomain but the short link redirected to admin page when opened, so I try to install YOURLS on my website root (https://mnazmiw.my.id), but I want YOURLS page opened if there is a GET variable (https://mnazmiw.my.id/?yourls), if there is none, the page will redirect to another page. So I tried to edit the index.php to:
`<?php
if(!isset($_GET['yourls'])) {
    header('Location: https://mnazmiw.my.id/portal');
};
include_once('admin/index.php')
?>`

The redirect works, but it turns out if I open a short link, it got redirected to the portal page too. (example: https://mnazmiw.my.id/li). 

Anyone knows how to fix it? Or even a better way to do it?

**Comment by ozh** [comment]
The problem is that when a short URL is requested, eg `sho.rt/blah` and `blah` isn't an existing file, it is turned into a GET var -- that's exactly what the .htaccess does. So basically you cannot do as you imagined.
